recipe/meta.yaml

```diff
@@ -11,21 +11,17 @@ source:
 
```

Comment:
While we're at it, maybe use the PyPI source at L8-L10 instead of GitHub? E.g.

```
source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omegaconf-{{ version }}.tar.gz
  sha256: 59ff9fba864ffbb5fb710b64e8a9ba37c68fa339a2e2bb4f1b648d6901552523
```

Generated using https://www.marcelotrevisani.com/grayskull

Reply:
Yes, good point.